COMMUNITY.md

```diff
@@ -0,0 +1,107 @@
+# Nephio Project Community membership
+
+This document describes the responsibilities and contributor roles in Nephio Project. 
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| New contributor |  - | Submits the Pull Request on GitHub | Anyone is welcome to participate |
+| Member | Active contributor | Sponsored by one reviewer and has several contributions to the project | Nephio GitHub org member|
+| Reviewer | Review contributions from other members | History of review and authorship in the project | [OWNERS] file reviewer entry |
+| Approver | Contributions acceptance approval| Highly experienced active reviewer and contributor to the project | [OWNERS] file approver entry|
+
+## New contributor
+
+Anyone is welcome to contribute to Nephio project. Existing members should make a point of being helpful 
+for new contributors in any way possible (PR workflow, finding relevant documentation, guiding with coding style etc.)
+Remember we were all '[new contributors]' once.
+
+## Member
+
+Members are active contributors in the community. Members might have PRs and GitHub issues assigned to them and are added to GitHub team(s).
+Prow's presubmit tests are automatically run for their PRs as well as GitHub Actions (where applicable). 
+Members are welcome to remain continuously active contributors to the community.
+
+### Requirements
+
+- Ensure affiliation is up to date in [openprofile.dev]. 
```

Comment:
Should this be a link? "[openprofile.dev]"

Reply:
Links are defined at the bottom of doc: https://github.com/radoslawc/nephio/blob/2f99de696abb492386973cdbd9441799d85c9385/COMMUNITY.md?plain=1#L107